pkg/option/config.go

```diff
@@ -1746,6 +1749,9 @@ type DaemonConfig struct {
 	// EnableWireguardUserspaceFallback enables the fallback to the userspace implementation
 	EnableWireguardUserspaceFallback bool
 
+	// WireguarPersistentKeepalive controls Wireguard PersistentKeepalive option.
+	WireguarPersistentKeepalive time.Duration
```

Comment:
Sorry I didn't catch it earlier, there is another typo here:

```suggestion
	WireguardPersistentKeepalive time.Duration
```

Reply:
ohhh, Thank you for the reminder.